index.js

```diff
@@ -83,10 +85,10 @@
 //. ```javascript
 //. //    add :: Number -> Number -> Number
 //. const add =
-//. def ('add')
-//.     ({})
-//.     ([$.Number, $.Number, $.Number])
-//.     (x => y => x + y);
+//. def ('add') // name of function
+//.     ({}) // type-class constraints
+//.     ([$.Number, $.Number, $.Number]) // function argument and output types
+//.     (x => y => x + y); // curried function implementation
```

Comment:
Would terser labels be equally informative, do you think?

```suggestion
//. def ('add')                             // name
//.     ({})                                // type-class constraints
//.     ([$.Number, $.Number, $.Number])    // input and output types
//.     (x => y => x + y);                  // implementation
```

Reply:
I think this really adds the clarity I intended, but more concisely.  I will have these changes pushed shortly.